plugin/trino-delta-lake/src/test/java/io/trino/plugin/deltalake/TestDeltaLakeBasic.java

```diff
@@ -635,6 +645,159 @@ private void testWriterAfterRenameColumnWithColumnMappingMode(String columnMappi
         assertUpdate("DROP TABLE " + tableName);
     }
 
+    /**
+     * @see deltalake.data_skipping_stats_columns
+     */
+    @Test
+    public void testDataSkippingStatsColumnsProperty()
+            throws Exception
+    {
+        String tableName = "test_data_skip_stats_columns_" + randomNameSuffix();
+        Path tableLocation = Files.createTempFile(tableName, null);
+        copyDirectoryContents(new File(Resources.getResource("deltalake/data_skipping_stats_columns").toURI()).toPath(), tableLocation);
+        assertUpdate("CALL system.register_table('%s', '%s', '%s')".formatted(getSession().getSchema().orElseThrow(), tableName, tableLocation.toUri()));
+
+        assertQuery(
+                "SELECT value FROM \"" + tableName + "$properties\" WHERE key = 'delta.dataSkippingStatsColumns'",
+                "VALUES 'lower,`UPPER`,`a.dot`,a.nested'");
```

Comment:
Could you add a helper method to return `delta.dataSkippingStatsColumns` property?
```java
assertThat(getDataSkippingStatsColumns(tableName)).isEqualTo("lower,`UPPER`,`a.dot`,a.nested");
...
private String getDataSkippingStatsColumns(String tableName)
{
    return (String) computeScalar("SELECT value FROM \"" + tableName + "$properties\" WHERE key = 'delta.dataSkippingStatsColumns'");
}
```